src/api/common/HttpClient.ts

```diff
@@ -0,0 +1,73 @@
+import log from 'loglevel';
+import type {
+    HubInfo,
+    RelayEstimation,
+    RelayOrDeployRequest
+} from 'src/common/relay.types';
+import type HttpWrapper from './HttpWrapper';
+
+const PATHS = {
+  GET_INFO: '/getaddr',
+  POST_REALY_REQEUST: '/relay',
+  POST_ESTIMATE: '/estimate'
+} as const;
+const VERIFIER_SUFFIX = '?verifier=';
+
+export default class HttpClient {
+  private readonly _httpWrapper: HttpWrapper;
+
+  constructor(httpWrapper: HttpWrapper) {
+    this._httpWrapper = httpWrapper;
+  }
+
+  async getHubInfo(relayUrl: string, verifier = ''): Promise<HubInfo> {
```

Comment:
I'm afraid that the name isn't really appropriate, because we're not retrieve the hub info only.
What about `getChainInfo`?

Reply:
done